lemarche/utils/emails.py

```diff
@@ -9,3 +14,20 @@ def whitelist_recipient_list(recipient_list):
     if settings.BITOUBI_ENV == "prod":
         return recipient_list
     return [email for email in recipient_list if (email and email.endswith("beta.gouv.fr"))]
+
+
+@task()
+def send_mail_async(
+    email_subject,
+    email_body,
+    recipient_list,
+    from_email=settings.DEFAULT_FROM_EMAIL,
+    fail_silently=False,
+):
+    send_mail(
+        subject=f"{EMAIL_SUBJECT_PREFIX} {email_subject}",
```

Comment:
```suggestion
        subject=f"{EMAIL_SUBJECT_PREFIX}{email_subject}",
```
pour éviter l'espace en prod ?